lib/oxidized/pfsense.rb

```diff
@@ -0,0 +1,25 @@
+class PfSense < Oxidized::Model
+  
+  comment  '# '
+  
+  #add a comment in the final conf
+  def add_comment comment
+    "\n###### #{comment} ######\n" 
+  end
+
+  cmd :all do |cfg|
+    cfg.each_line.to_a[1..-2].join
+  end
+  
+  #show the persistent configuration
+  pre do
```

Comment:
@stokbaek care to explain why this model is so different?
couldn't we just follow the others with something like:

``` ruby
cmd 'cat /cf/conf/config.xml'
```